code/modules/mob/living/carbon/human/species/necromorph/mob.dm

```diff
@@ -5,6 +5,7 @@
 `	I am well aware this is not how human mobs and species are supposed to be used
 */
 /mob/living/carbon/human/necromorph
+	psychosis_immune = TRUE //Because..y'know.
```

Comment:
This should be set via species, not directly on the mob.
Usage of necromorph mob subtypes is temporary, everything works through species code